pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -4017,7 +4023,7 @@
         "4.0.0",
         "4.3.0"
       ],
-      "BaselineVersion": "4.3.0",
+      "BaselineVersion": "5.0.0",
```

Comment:
@ericstj is it expected this would've been 4.3.0?

Reply:
Yes, this needs to be undone.  System.Runtime.Loader no longer builds in master.